woof/tomato-attack.html

```diff
@@ -0,0 +1,83 @@
+	</div>
+	<div class="block">
+		<p>1) Set the backdrop for your game and create your tomato sprite.  </p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step1.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>2) Create another sprite to shoot the tomato out of. Place the tomato at the shooting sprite's position. </p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step2.html">Play this demo in full screen.</a>
+	</div>
+
+	<div class="block">
+		<p>3) Have the shooting sprite move left and right with the arrow keys (or other keys). Hide the tomato for now. </p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step3.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>4) When a key is pressed (the up arrow in the demo), make the tomato shoot out of the shooting sprite. When it reaches the top of the screen, move it back to the shooting sprite and hide it again.</p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step4.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>5) Create a veggie sprite and place it in a random position. </p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step5.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>6) Create more veggie sprites and make them constantly appear at random times in random places. Have them appear for only short intervals of time.</p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step6.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>7) Create different costumes for the veggie sprites and have each veggie wear a random costume when it appears.</p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step7.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>8) If a tomato hits a veggie, make the veggie and tomato disappear and send the tomato back to the shooting sprite's position.</p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step8.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>9) Create a score variable and display it on the screen. </p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step9.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>10) Every time the tomato hits a veggie, increase the score.</p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step10.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>11) Create a lives variable and display it on the screen.</p>
+		<br><a target="_blank" href="./snippets/tomato-attack-step11.html">Play this demo in full screen.</a>
+	</div>
+	
+	<div class="block">
+		<p>12) Every time a tomato is fired and it doesn't hit a veggie, decrease the lives variable. Display a message explaining why the player lost a life for a few seconds.</p>
```

Comment:
Maybe make this a little more explicit? "Every time a tomato reaches the top of the screen without hitting a veggie..."